### DISCUSSION TITLE
Is it possible to create page for locale manually? (asking because of MDX)

### DISCUSSION THREAD

**Discussion by cibulka** [vercel/next.js#32102] [Help] [answered]
Hi everyone! 

Is it possible to create locale version of a page manually? 

Say I have a page `/info` with two language versions (en and cs). I need my links (https://example.com/en/info and https://example.com/cs/info) resolve to two different documents in my `/pages` folder. 

## Why am I asking

I've started with MDX today by [this guide](https://nextjs.org/docs/advanced-features/using-mdx) and it is **very cool**. As I understand the guide though, it is meant to be used only at the page level. This is edited example from the guide that sits on `pages/info.mdx`:

```
import { MyLayoutComponent } from 'my-components'

# English version of INFO page

This is a list in markdown:

- One
- Two
- Three

export default = ({ children }) => <MyLayoutComponent>{children}</MyLayoutComponent>
```

How should I proceed when I need the czech version of this page as well? Is creating manual locale versions of the pages the way or would you suggest another approach altogether?

Let me know if the question makes sense. Thanks in advance, P!

**Comment by wwwdepot** [comment, marked as answer]
Yes with [with-mdx-remote](https://github.com/vercel/next.js/tree/canary/examples/with-mdx-remote) example.

You have `pages/info.js` instead of `pages/info.mdx`. Idea is using `locale` from `getStaticProps` to read `info.${locale}.mdx` and render them.

**Reply by cibulka** [reply]
Thank you, that pointed me in the right direction!
